vscode-trace-extension/src/extension.ts

```diff
@@ -50,11 +50,23 @@ export function activate(context: vscode.ExtensionContext): ExternalAPI {
         }
     }));
 
+    const updateUris = async (url?: string): Promise<void> => {
+        url = url ?? getTraceServerUrl();
+        const baseUri = vscode.Uri.parse(url);
+        const extUri = await vscode.env.asExternalUri(baseUri);
+        const extUriString = extUri.toString();
+
+        tracesProvider.updateTraceServerUrl(extUriString);
+        myAnalysisProvider.updateTraceServerUrl(extUriString);
+        TraceViewerPanel.updateTraceServerUrl(extUriString);
+    };
+
```

Comment:
@williamsyang-work a question, what if a user executes the command `Trace Server: Start (if stopped)` manually using the command palette, shouildn't `updateUris()` be called as well? Maybe when command `serverStatus.started` is executed? Please let me know.

Reply:
@bhufmann I tested starting and stopping the trace server and it worked as-is for me.
My understanding is: as long as server url remains the same we don't need to update it.  The process on that port can stop, start, restart, etc.

Reply:
@williamsyang-work When connecting to the remote host and executing the command `Trace Server: Start (if stopped)` then the server is started, but the port forwarding is not working because `updateUris()` and hence`vscode.asExternalUrl` is not executed. Start and stop is still working in this case. I tested that case with a remote running in a virtual machine and I saw it not working.

However, when using the open trace command that starts the server and `updateUris()` is executed will enable port forwarding for the manual start command.

I think we need to address the manual start case as well.

Reply:
Per our discussion `updateUris()` is now called in the `serverStatus.started` command.  This updates the webview and starts port forwarding.  

There seems to be another bug where it shows the server's data accurately, but then changes back to 'No Traces Open: Open Trace' state after some time.